cli/azd/pkg/project/framework_service_docker.go

```diff
@@ -49,5 +57,38 @@ func NewDockerProject(config *ServiceConfig, env *environment.Environment, docke
 		env:       env,
 		docker:    docker,
 		framework: framework,
+		options:   createDockerOptions(config),
+	}
+}
+
+func createDockerOptions(config *ServiceConfig) DockerProjectOptions {
+	dockerOptions := DockerProjectOptions{
+		Path:     "./Dockerfile",
+		Platform: "amd64",
+		Context:  ".",
+	}
+
+	if len(config.Options) == 0 {
+		return dockerOptions
+	}
+
+	dockerMap, ok := config.Options["docker"]
+	if !ok {
+		return dockerOptions
+	}
+
+	log.Printf("found custom docker options %s\n", dockerMap)
+
+	jsonBytes, err := json.Marshal(dockerMap)
+	if err != nil {
+		log.Printf("error marshalling project options to JSON: %s", err.Error())
+		return dockerOptions
+	}
+
+	if err := json.Unmarshal(jsonBytes, &dockerOptions); err != nil {
```

Comment:
It would be nice to print a warning to list any `ignored options`.  For example, right now we are adding support for path, platform and Context.  If in the future, there is a new option and customers try to override it with azd, azd would mention that the option is still unknown to azd. Then customers can submit an issue or create a PR to add it

Reply:
I've updated the JSON schema which should warn the user of unsupported options.  This should be good enough for now.